File: CS/Models/DocumentPreviewDialogViewModel.cs
using DevExpress.Mvvm;
using DevExpress.Utils;
using DevExpress.XtraReports.UI;

namespace ShowReportPreview_MVVM.Models {
    public class DocumentPreviewDialogViewModel : ViewModelBase {
        public XtraReport Report { get; }
        public DocumentPreviewDialogViewModel(XtraReport report) {
            Report = report;
        }
    }
}

File: VB/Models/DocumentPreviewDialogViewModel.vb
Imports DevExpress.Mvvm
Imports DevExpress.Utils
Imports DevExpress.XtraReports.UI

Namespace ShowReportPreview_MVVM.Models
	Public Class DocumentPreviewDialogViewModel
		Inherits ViewModelBase

		Public ReadOnly Property Report() As XtraReport
		Public Sub New(ByVal report As XtraReport)
			Me.Report = report
		End Sub
	End Class
End Namespace
